src/sentry/api/endpoints/project_keys.py

```diff
@@ -51,9 +51,8 @@ def get(self, request: Request, project) -> Response:
         """
         Return a list of client keys bound to a project.
         """
-        queryset = ProjectKey.objects.filter(
-            project=project, roles=F("roles").bitor(ProjectKey.roles.store)
-        )
+        objects = ProjectKey.objects if request.user.is_superuser else ProjectKey.objects.user()
```

Comment:
Usually we also check that the user has an active superuser/staff session as well.

Reply:
Thanks, changed to `is_active_superuser` now.